docs/CODEOWNERS

```diff
@@ -0,0 +1,138 @@
+# Archipelago World Code Owners / Maintainers File
+#
+# This file is used to notate who the current "owners" or "maintainers" of any currently merged world folder. For any
+# pull requests that modify contents of these worlds, a code owner must approve the PR in addition to a core maintainer.
+# This is not to be used for files/folders outside the /world folder, those will always need sign off from a core
+# maintainer. All usernames must be GitHub usernames (and are case sensitive).
+
+# Adventure
+/worlds/adventure/ @JusticePS
+
+# A Link to the Past
+/worlds/alttp/ @Berserker66
+
+# ArchipIDLE
+/worlds/archipidle/ @LegendaryLinux
+
+# Sudoku (BK Sudoku)
+/worlds/bk_sudoku/ @Jarno458
+
+# Blasphemous
+/worlds/blasphemous/ @TRPG0
+
+# ChecksFinder
+/worlds/checksfinder/ @jonloveslegos
+
+# Clique
+/worlds/clique/ @ThePhar
+
+# Dark Souls III
+/worlds/dark_souls_3/ @Marechal-L
+
+# Donkey Kong Country 3
+/worlds/dkc3/ @PoryGone
+
+# DLCQuest
+/worlds/dlcquest/ @axe-y @agilbert1412
+
+# Factorio
+/worlds/factorio/ @Berserker66
+
+# Final Fantasy
+/worlds/ff1/ @jtoyoda
+
+# Hollow Knight
+/worlds/hk/ @dewiniaid
+
+# Hylics 2
+/worlds/hylics2/ @TRPG0
+
+# Kingdom Hearts 2
+/worlds/kh2/ @JaredWeakStrike
+
+# Links Awakening DX
+/worlds/ladx/ @zig-for
+
+# Lufia II Ancient Cave
+/worlds/lufia2ac/ @el-u
```

Comment:
Ludwig is also listed as a maintainer on the discord, but again unsure if they have a github account since i dont see them in the commit history

Reply:
Need their github username. :P

Reply:
Done.